src/styles/index.scss

```diff
@@ -281,20 +261,24 @@ button {
 
 .cook-recipe-button,
```

Comment:
The cook-recipe-button is sticking out of the sidebar because it is assigned to that width of 257px. Is there a way for them to have the same properties except the width?

Reply:
Hi Natalia - thank you for reviewing this PR. I made the changes you suggested - thank you :)